### PR TITLE
feat(oauth2): implement login via google

### PR DESCRIPTION
**Pull request by petrjasek**
SDESK-1268

### CODE REVIEW

**Comment by petrjasek** [comment]
requires https://github.com/superdesk/superdesk-core/pull/924

File: superdesk.config.js [hunk 1]
```diff
@@ -35,8 +35,9 @@ module.exports = function(grunt) {
             }
         },
         features: {
-            swimlane: {columnsLimit: 4}
+            swimlane: {columnsLimit: 4},
```
Comment (gbbr) [inline superdesk.config.js]: Forgot a comma

File: scripts/core/auth/login-modal.html [hunk 2]
```diff
@@ -23,12 +23,19 @@
                         </div>
                     </button>
                 </fieldset>
+
+                <fieldset class="buttons" ng-show="allowedAuth.google">
+                    <button class="btn btn--sd-green"
+                        ng-click="openLoginPopup('google')" translate>Log In with Google</button>
```
Comment (gbbr) [inline scripts/core/auth/login-modal.html]: There was a comment from Vince about the consistency of this label. What if we use `{{ 'Log In' | translate }} with Google` as above for now?

Comment (petrjasek) [inline scripts/core/auth/login-modal.html]: vincers comment was about `Login` which was there before, `Log In` is updated version

**Pull request merged**